src/features/search/components/SearchResultsContent/SearchResultsContent.native.test.tsx

```diff
@@ -24,6 +23,7 @@ import { GeolocPermissionState, Position } from 'libs/location'
 import { LocationMode } from 'libs/location/types'
 import { SuggestedPlace } from 'libs/place/types'
 import { mockedSuggestedVenue } from 'libs/venue/fixtures/mockedSuggestedVenues'
+import { useVenuesInRegionQuery } from 'queries/useVenuesInRegionQuery/useVenuesInRegionQuery'
```

Comment:
il me semble qu'on s'était dit que le dossier était nommé comme la feature

```suggestion
import { useVenuesInRegionQuery } from 'queries/venueMap/useVenuesInRegionQuery'
```